lib/mix/tasks/nerves/new.ex

```diff
@@ -87,17 +87,17 @@ defmodule Mix.Tasks.Nerves.New do
 
       mix nerves.new blinky --module Blinky
 
-  Generate a project configured to use `nerves_init_gadget`
-
-      mix nerves.new blinky --init-gadget
-
   Generate a project that only supports Raspberry Pi 3
 
       mix nerves.new blinky --target rpi3
 
   Generate a project that supports Raspberry Pi 3 and Raspberry Pi Zero
 
       mix nerves.new blinky --target rpi3 --target rpi0
+
+  Generate a project without `nerves_init_gadget`
```

Comment:
Maybe add the word minimal so people don't get the wrong idea with what's init?  I.e. "Generate a minimal project without `nerves_init_gadget`"?

Reply:
Should we always refer as a `minimal` project only in the last example.